### DISCUSSION TITLE
Resources to endpoint

### DISCUSSION THREAD

**Discussion by marioidival** [hermeznetwork/price-updater-service#2] [Ideas]
Hello guys! 
Today there are two endpoint's to return `prices` on hermez-node, the currencies and tokens.

What you think merge this two endpoints into another called prices (as is there here) and add an argument on URI

Example:
```txt
hermez-node:
/currenices
/tokens

price updater
/prices/tokens
/prices/currencies
```

**Comment by marioidival** [comment]
@jeffprestes @mfcastellani @Mikelle @tclemos

**Reply by jeffprestes** [reply]
I think it's not necessary because the service is focused on price. But what I miss is API version.
Maybe /v1/currencies and /v1/tokens
